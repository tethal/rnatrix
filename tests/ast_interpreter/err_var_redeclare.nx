fun main() {
    var x = 10;
    var x = 20;
}
