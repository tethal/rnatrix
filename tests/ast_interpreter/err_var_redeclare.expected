<string>:2:5: error: variable "x" already defined
var x = 20;
    ^